docs/modules/ROOT/pages/Case_Study_Mill_vs_Gradle.adoc

```diff
@@ -0,0 +1,201 @@
+= Case Study: Mill vs Gradle
+
+++++
+<script>
+gtag('config', 'AW-16649289906');
+</script>
+++++
+
+This page compares using Mill to Gradle, using the https://github.com/mockito/mockito[Mockito Testing Library]
+codebase as the example. Netty is a medium sized codebase, 100,000 lines of Java split over 22
+subprojects. By porting it to Mill, this case study should give you
+an idea of how Mill compares to Maven in more realistic, real-world projects.
+
+To do this, we have written a Mill `build.sc` file for the Mockito project. This can be used
+with Mill to build and test the various submodules of the Mockito project without needing to
+change any other files in the repository:
+
+- https://github.com/com-lihaoyi/mill/blob/main/example/thirdparty/mockity/build.sc[Mockito build.sc file]
+
+== Completeness
+
+The Mill build for Mockito is not 100% complete, but it covers most of the major parts of Mockito:
+compiling Java, running JUnit tests. For now, the Android, Kotlin, and OSGI tests are skipped
+since Mill does not support those frameworks.
```

Comment:
I'll tweak the wording. We can include them in this example, but I'd like to in-source them as the mill `contrib/` modules first, to avoid having our test suite depend too much on downstream github projects.